### ISSUE TITLE
Download videos and other resources too?

### ISSUE THREAD

**Issue by JohnVeness** [EugeneLoy/edx-archive#27]
I know that you reference edx-dl in the readme, but have you considered edx-archive downloading videos, PDFs, and other resource files too?

**Comment by EugeneLoy** [comment]
Yes, there is a great chance of this being implemented sometime in the future.

At this point I would like to direct effort towards supporting multiple platforms and providing useful features that are not implemented by `edx-dl`, `coursera-dl` and alike.

Once this is done I would, probably, start adding support for videos\resources, so user won't have to juggle with multiple tools to build complete archive of the course.
